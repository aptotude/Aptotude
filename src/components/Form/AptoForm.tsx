export { Form as AptoForm } from 'semantic-ui-react';
import 'semantic-ui-css/semantic.min.css';
