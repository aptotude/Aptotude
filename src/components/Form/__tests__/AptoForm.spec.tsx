import React from 'react';
import { render, cleanup } from 'react-testing-library';
import { AptoForm } from '../AptoForm';

const Form = () => (
  <div>
    <AptoForm.Field required>
      <label>Name</label>
      <AptoForm.Input />
    </AptoForm.Field>
  </div>
);

describe('AptoForm', () => {
  afterEach(cleanup);

  it('makes fields required', () => {
    const { asFragment } = render(<Form />);
    expect(asFragment()).toMatchSnapshot();
  });
});
