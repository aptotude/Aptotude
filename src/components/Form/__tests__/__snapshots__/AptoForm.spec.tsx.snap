// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AptoForm makes fields required 1`] = `
<DocumentFragment>
  <div>
    <div
      class="required field"
    >
      <label>
        Name
      </label>
      <div
        class="field"
      >
        <div
          class="ui input"
        >
          <input
            type="text"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
